website_and_docs/content/documentation/webdriver/bidi_apis.en.md

```diff
@@ -5,19 +5,56 @@ weight: 11
 aliases: ["/documentation/en/webdriver/bidi_apis/"]
 ---
 
-In Selenium 4, new Evented APIs were introduced that
-allow users to be able to capture events from the
-browser as they happen rather than using the
-traditional approach of Request/Response that
-WebDriver has used for other APIs.
+Selenium is working with Browser Vendors to create the 
+[BiDirectional WebDriver Protocol](https://w3c.github.io/webdriver-bidi/) 
+as a means to provide a stable, cross-browser implementation of a subset of the functionality found
```

Comment:
The goal isn't to provide a subset of the CDP. The goal is to provide a stable, cross-browser implementation of a bi-directional API designed for browser automation, with testing a specific use case. We want nothing to do with debugging.

Reply:
Yes, this was the part I was getting handwavey at. What about:

> Selenium is working with browser vendors to create the 
[BiDirectional WebDriver Protocol](https://w3c.github.io/webdriver-bidi/) 
as a means to provide a stable, cross-browser API for leveraging the bidirectional
functionality useful for both browser automation generally and testing specifically. 
Before now, users seeking this functionality have had to rely on 
the [Chrome DevTools Protocol](../../support_packages/chrome_devtools), 
with all of its frustrations and limitations.

Reply:
Sounds good, with two minor changes: it's "WebDriver Bidi" not "Bidi WebDriver", and prefer "API that uses" to "for leveraging". I always feel that words like "leverage" are _very_ corporate.